app/views/state_file/questions/data_transfer_offboarding/edit.html.erb

```diff
@@ -13,11 +13,13 @@
     <p><%= vita_introduction %></p>
   <% end %>
 
-  <% vita_link = I18n.locale == :en ? StateFile::StateInformationService.vita_link_en(current_state_code) : StateFile::StateInformationService.vita_link_es(current_state_code) %>
-  <% if vita_link.present? %>
-    <p><%= link_to t("state_file.questions.eligible.faq_link"), state_faq_path(us_state: current_state_code) %></p>
-    <%= link_to(vita_link, class: "button button--primary button--wide spacing-below-15", target: "_blank", rel: "noopener noreferrer") do %>
-      <%= t('state_file.questions.eligible.vita_option.connect_to_vita') %>
+  <% if current_state_code == "nj" %>
```

Comment:
[dust] For this case and the other one, I wonder if instead of checking for a state code or for the link itself, you could move this content to the yml files as something like nj_data_transfer_offboarding_links (or split it up as needed) and pass the vita link as a param?

Reply:
@mmazanec22 I moved the content to yml as you suggested and I do think I like it better now. Thoughts?